packages/pluggableWidgets/slider-web/src/utils/createHandleGenerator.tsx

```diff
@@ -0,0 +1,50 @@
+import { createElement } from "react";
+import { SliderContainerProps } from "../../typings/SliderProps";
+import Tooltip from "rc-tooltip";
+import { Handle } from "rc-slider";
+
+// Copied from https://github.com/react-component/slider/blob/8.6.6/src/Slider.jsx#L165
+interface HandleGeneratorProps {
+    className: string;
+    prefixCls?: string;
+    vertical: boolean;
+    offset: number;
+    value: number;
+    dragging?: boolean;
+    disabled?: boolean;
+    min?: number;
+    max?: number;
+    index: number;
+    tabIndex?: number;
+    style?: React.CSSProperties;
+    ref?: React.Ref<any>;
+}
+
+type HandleGenerator = (props: HandleGeneratorProps) => JSX.Element;
+
+export function createHandleGenerator(props: SliderContainerProps): HandleGenerator | undefined {
+    const { tooltip, showTooltip } = props;
+
+    if (!showTooltip) {
+        return;
+    }
+
+    return function handleGenerator(generatorProps: HandleGeneratorProps): JSX.Element {
+        const { dragging, index, ...restProps } = generatorProps;
+        const overlay = <div>{tooltip?.value}</div>;
```

Comment:
This means that if the tooltip value is still loading (cuz it's a `DynamicValue`), we just show an empty tooltip or even "undefined". Is this according to PM requirements? If not, can you check. If so, can you check whether this doesn't show a tooltip with "undefined"?

If you need to check on status, you can use `ValueStatus` from `mendix`

Reply:
Can be solved with:
```
<div>{tooltip?.value ?? ""}</div>;
```